### PR TITLE
Portfolio: Added mobile menu

### PR DESCRIPTION
**Pull request by amare1990**
- Add ..eslintrc.json file.
- Add markup page for the mobile menu.
- Add CSS styles for the mobile menu.
- Add JS for adding dynamics to the webpage.
- Add scroll events for each menu link.
- Modify the homepage of the mobile version to be blurred

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  
+}
+
+hamburger.addEventListener('click', mobileMenuDisplay);
+closeButton.addEventListener('click', mobileMenuDisplay);
+
+const close_window = document.querySelector('.mobile-menu-close').addEventListener('click', function(){
+  close_window.style.display = 'none';
+  
+});
```
Comment (victorokoroji) [inline js/scripts.js]: - [ ] You have really tried implementing the mobile menu functionality 👍🏽, but currently, the function you added to toggle the mobile menu is not working. When I click the hamburger, the mobile menu doesn't display but rather I get an error in the console. Kindly debug the issue and fix it 👍🏽

image

File: js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  else {
+      mobileMenuContainer.style.display = 'none';
+      icon.classList.replace('fa-times', 'fa-bars');
+  }
+  //wrapperMain.style.display = 'none';
+  
+}
+
+hamburger.addEventListener('click', mobileMenuDisplay);
+closeButton.addEventListener('click', mobileMenuDisplay);
```
Comment (vickymarz) [inline js/scripts.js:29]: - [ ] You have really tried implementing the mobile menu functionality 👍🏽, but currently, the function you added to toggle the mobile menu is not working. When I click the hamburger, the mobile menu doesn't display but rather I get an error in the console. Kindly debug the issue and fix it 👍🏽

image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @amare1990 , 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css [hunk 3]
```diff
@@ -660,6 +650,113 @@ form {
...
+  mix-blend-mode: multiply;
+  z-index: 100;
+  display: none;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  margin-top: 80px;
+  backdrop-filter: blur;
```
Comment (FrederickMih) [inline css/styles.css]: - Great job for fixing the hamburger button issue raised by the previous reviewer :+1: 

- [ ] Your design is slightly different from the one in Figma. Kindly check the screenshots below:

| Yours | Figma |
| --- | --- |
| Screenshot from 2022-09-13 22-53-07 | Screenshot from 2022-09-13 23-00-34 |

Let's make sure that the font weight, and size are the same as in Figma. Also, adjust your close icon to position as seen in the original screenshot. Your contact option is hidden by the header text. Let's fix the issues. Also, ensure that there are no console errors.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @amare1990 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- The Hamburger button works as expected :+1: 

 ## Required Changes ♻️


_Check the comments under the review._

## Optional suggestions

- [Optional] **Please kindly make sure to include a descriptive README file. I think the previous reviewer forgot to invalidate the PR. We required a descriptive README.md file for each project.**


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smunozmo — APPROVED** [review]
Hi @amare1990, 👋 

## Highlights 👏 

- Excellent job making all the suggested changes! 🎉 
- Mobile menu is working properly, perfect! 😎 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**